draft-ietf-httpbis-binary-message.md

```diff
@@ -571,7 +565,14 @@ versions.  In particular, this format does not allow for:
 * framing of responses that depends on the corresponding request (such as HEAD)
   or the value of the status code (such as 204 or 304)
 
-Many of these same restrictions are shared by HTTP/2 {{H2}} and HTTP/3 {{H3}}.
+This format shares some of these properties with HTTP/2 and HTTP/3.
```

Comment:
This sentence now reads a bit strangely in combination with the list above. In simple reading it goes "this format lacks X, Y, Z that are defined for H2 and H3 ... this format shares some of the features we just said are lacking". 

Perhaps rephrase this as "some of these features are similarly not supported by HTTP/2 and HTTP/3, making the features unique to HTTP/1.1"?